Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Favorites/AssetBrowserFavoritesView.cpp

```diff
@@ -0,0 +1,179 @@
+            , m_delegate(new FavoritesEntryDelegate(this))
+        {
+            setModel(m_favoritesModel.data());
+            setItemDelegate(m_delegate.data());
+            setSelectionMode(QAbstractItemView::SelectionMode::SingleSelection);
+            setContextMenuPolicy(Qt::CustomContextMenu);
+            setMinimumHeight(MinimumHeight);
+            setDragEnabled(true);
+            setAcceptDrops(true);
+            setDragDropMode(QAbstractItemView::DragDrop);
+            setDropIndicatorShown(true);
+            setContextMenuPolicy(Qt::CustomContextMenu);
+
+            m_favoritesModel->SetParentView(this);
+
+            m_delegate.data()->SetShowFavoriteIcons(true);
+
+            header()->hide();
+
+            connect(m_delegate.data(), &EntryDelegate::RenameEntry, this, &AssetBrowserFavoritesView::AfterRename);
+            connect(this, &QTreeView::customContextMenuRequested, this, &AssetBrowserFavoritesView::OnContextMenu);
+        }
+
+        void AssetBrowserFavoritesView::SetSearchWidget(SearchWidget* searchWidget)
+        {
+            m_favoritesModel.data()->SetSearchWidget(searchWidget);
+        }
+
+        void AssetBrowserFavoritesView::OnContextMenu([[maybe_unused]] const QPoint& point)
+        {
+            auto selectedAssets = selectedIndexes();
+
+            if (selectedAssets.isEmpty())
+            {
+                return;
+            }
+
+            QModelIndex selected = selectedAssets.at(0);
+
+            if (!selected.parent().isValid())
+            {
+                // This is the "favorites" folder.
+                return;
+            }
+
+            AssetBrowserFavoriteItem* favoriteItem = static_cast<AssetBrowserFavoriteItem*>(selected.internalPointer());
+
+            AssetBrowserFavoriteItem::FavoriteType favoriteType = favoriteItem->GetFavoriteType();
+            QMenu menu(this);
+
+            if (favoriteType == AssetBrowserFavoriteItem::FavoriteType::AssetBrowserEntry)
+            {
+                AZStd::vector<const AssetBrowserEntry*> selectedEntries;
+
+                EntryAssetBrowserFavoriteItem* favoriteEntry = static_cast<EntryAssetBrowserFavoriteItem*>(favoriteItem);
+                selectedEntries.push_back(favoriteEntry->GetEntry());
+
+                AssetBrowserInteractionNotificationBus::Broadcast(
+                    &AssetBrowserInteractionNotificationBus::Events::AddContextMenuActions, this, &menu, selectedEntries);
+            }
+            else
+            {
+                if (!m_favoritesModel.data()->GetIsSearchDisabled())
+                {
+                    menu.addAction(
+                        QObject::tr("Apply to Asset Browser"),
+                        [this, favoriteItem]()
+                        {
+                            m_favoritesModel.data()->Select(favoriteItem);
+                        });
+                    menu.addAction(
+                        QObject::tr("Update"),
```

Comment:
nit: "Update Saved Search" would be more clear here

Reply:
Good idea, done.